tests/ops/op_math/test_linear_combination.py

```diff
@@ -705,50 +730,11 @@ def test_simplify_while_queueing(self):
         # check that the simplified LinearCombination is in the queue
         assert q.get_info(H) is not None
 
-    def test_data(self):
-        """Tests the obs_data method"""
-        # pylint: disable=protected-access
-
-        H = qml.ops.LinearCombination(
-            [1, 1, 0.5],
-            [Z(0), Z(0) @ X(1), X(2) @ qml.Identity(1)],
-        )
-        data = H._obs_data()
-
-        expected = {
-            (0.5, frozenset({("Prod", qml.wires.Wires([2, 1]), ())})),
-            (1.0, frozenset({("PauliZ", qml.wires.Wires(0), ())})),
-            (1.0, frozenset({("Prod", qml.wires.Wires([0, 1]), ())})),
-        }
-
-        assert data == expected
-
-    def test_data_gell_mann(self):
```

Comment:
I have to admit I hadn't thought about this before - qutrit operator arithmetic is also all covered by our new approach?

Reply:
I dont think we removed any qutrit related tests so I would assume yes. This test here that I remove is just about the `_obs_data`